content/blog/open-source-monitoring-at-the-hpe-customer-technology-center-böblingen.md

```diff
@@ -0,0 +1,377 @@
+
+First, the exporter is reading the input data, i.e. the ILO ip addresses and the ILO username and password. Next the exporter is retrieving the correct URLs for each monitored server and store it in a Python directory.  Afterwards, the script starts the http server and the corresponding Prometheus counters and gauges. Having completed these preparation steps, the script enters an endless loop that first captures the start time of the current iteration, second retrieves the data from the monitored server, third displays the captured data and waits until the current monitoring intervall is completed. Additionally, I have added that at least once per day the input data and monitor URLs are refreshed. This allowed me to change the list of monitored systems without starting and stopping the script. I just needed to edit the input data file to adjust the list of monitored systems.
```

Comment:
Because you're talking about something that could happen now (because it is a function of what it does), I can see the use of present tense here. But keep it simple. For example, the first part would read better as:

"First, the exporter reads the input data; i.e. the ILO IP addresses and the ILO username and password. Next, the exporter retrives the correct URLs for each monitored server and stores it in a Python directory. Afterwards, the script starts the http server..."

Everything else follows along with that method, so it works, just about until the second to the last sentence, which I would change to:

"This allows me to change the list of monitored systems without starting and stopping the script. I just need to edit the input data file to adjust the list of monitored systems."